src/keyword_vpd_parser.cpp

```diff
@@ -48,11 +50,14 @@ types::VPDMapVariant KeywordVpdParser::parse()
             throw(DataException("Invalid Small resource type End"));
         }
 
+        validateChecksum(checkSumStart, checkSumEnd);
```

Comment:
move the incrementing two bytes out of validateChecksum, so it is visible incrementing two bytes to check the  VPD end Tag.

Reply:
I agree, validate should only validate. Also could you please update the parameters as per the guidelines?

Reply:
okay